lib/suspenders/version.rb

```diff
@@ -4,5 +4,5 @@ module Suspenders
     .read("#{File.dirname(__FILE__)}/../../.ruby-version")
     .strip
     .freeze
-  VERSION = "1.55.1".freeze
+  VERSION = "1.55.2".freeze
```

Comment:
Can we undo this version bump? There are other unreleased changes in mainline and we may want to version them differently. Thank you!